implementations.md

```diff
@@ -0,0 +1,13 @@
+# Implementations of Container Specification
+
+## Runtime
+
+* [opencontainers/runc](https://github.com/opencontainers/runc) - reference implementation of OCF runtime
+* [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCF
+* [huawei-openlab/ocp-testing](https://github.com/huawei-openlab/ocp-testing) - Testing framework for OCF runtime
```

Comment:
I'd put this in a separate section (`## Testing`?), since it's not an implementation of the runtime.